pwiz_tools/Skyline/Util/UtilIO.cs

```diff
@@ -843,6 +843,25 @@ public static bool IsFile(string path)
             return !IsDirectory(path);
         }
 
+        public static bool IsWriteLocked(string path)

```

Comment:
Usually "write locked" would mean that no one can read the file.
This method seems to be checking whether it's possible to write to the file, so a more appropriate name would be "IsReadLocked" or maybe "IsLocked" or "IsUnwritable"

Reply:
I guess I see your point. I prefer a name that doesn't require a ! For use for probably IsWritable is better than IsUnwritable or IsNotWitable. And correct use of IsReadLocked seems confusing.